temporalcli/client.go

```diff
@@ -142,6 +147,27 @@ func payloadCodecInterceptor(namespace, codecEndpoint, codecAuth string) (grpc.U
 	)
 }
 
+func errorInterceptor() grpc.UnaryClientInterceptor {
```

Comment:
Unused

Reply:
I'd wondered what all would shake out from deleting `dialGRPC`. Thanks for catching these